src/renderers/shaders/ShaderChunk/envmap_pars_fragment.glsl

```diff
@@ -4,6 +4,11 @@
 #endif
 
 #ifdef USE_ENVMAP
+
+	#if defined( USE_BUMPMAP ) || defined( USE_NORMALMAP ) || defined( PHONG )
```

Comment:
Yes, finally found it. It includes all `envmap_*` except `envmap_fragment.glsl`. In this case above change should be safe and everything will be good.